docker/docker/src/main/java/org/arquillian/cube/docker/impl/await/PollingAwaitStrategy.java

```diff
@@ -116,40 +131,66 @@ public boolean await() {
     }
 
     private boolean executeWaitForIt(String containerIp, int port) {
-        if (! new File(WAIT_FOR_IT_SCRIPT_DIRECTORY).exists()) {
-            if (! new File(WAIT_FOR_IT_SCRIPT_DIRECTORY).mkdirs()) {
-                throw new IllegalArgumentException("Couldn't create the Arquillian directory at ~/.arquillian");
+
+        // We copy our wait-for-it-sh.sh file form classpath
+        // Our wait-for-it-sh.sh script also works with busybox/alpine which is not true with the official one.
```

Comment:
In theory not, since the project is not maintained anymore according to comments and PR in queue.